stdlib/sampledata/sampledata.flux

```diff
@@ -0,0 +1,310 @@
+// Package sampledata provides functions that return basic sample datasets.
+package sampledata
+
+
+import "csv"
+
+option start = 2021-01-01T00:00:00Z
+option stop = 2021-01-01T00:01:00Z
```

Comment:
These options are not used within this package, what do they do?

Reply:
What does changing them do? I don't see any use of them within the package so I am confused what they do. Yes a comment would be great as I am not quite understanding their purpose.

Reply:
I guess they could just be made into regular variables, just by removing the `option` keyword?